cps/main.c

```diff
@@ -71,6 +71,7 @@ get_cps_conf(void)
 static int
 cleanup_cps(void)
 {
+	route_event_sock_close(&cps_conf);
```

Comment:
You may want to add `route_event_sock_close()` to the comment below.

Comment below copied here to help:
```/* rte_kni_release() and rte_acl_free() can be passed NULL. */```